src/js/ui/menu.js

```diff
@@ -304,6 +310,11 @@ Menu.prototype.selection = function(callback) {
   simply.impl.menuSelection();
 };
 
+Menu.prototype.setSelection = function(sectionIndex, itemIndex) {
+  this._selection = { sectionIndex: sectionIndex, itemIndex: itemIndex };
+  this._select();
+};
+
```

Comment:
Sorry about that, it was not documented since its API is not set in stone.

`selection` will query the current menu selection. Since the pebble does not actively transmit the current selection, the current selection is unknown by JS until it is queried. In order to let the developer be able to know when the selection is known right away, it takes a callback.

```
// Get the selection
menu.selection(function(e) {
  console.log(e.sectionIndex, e.itemIndex);
});
```

The proposal then is to be able to change the selection:
```
menu.selection(0, 1);
```

The underlying `simply.impl.menuSelection` function is already equipped to handle the above API.

Most of the classes have accessor methods that are both getters and setters. `item`, `items`, and `section` work the same way, except they are synchronous. `selection` isn't synchronous when getting since when the user scrolls through the menu, events are not being created so as to not congest the bluetooth connection and incur more bluetooth costs.

Hmm, sure, you can rename `selection` to `selected` as well. It looks like the C API goes with that naming. You can rename the underlying state and variables to use selected as well, or leave them and I can rename them later.

Lastly, the documentation should be in a separate PR since master is used to replicate the docs at pebble.com. If you don't have the time to write docs, you can also leave it out and I'll document it when pebble.js is updated on CloudPebble.